.github/workflows/ci.yml

```
@@ -5,7 +5,9 @@ on: # yamllint disable-line rule:truthy
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    branches:
+      - main
+      - 'feature/datavis'
```

we don't need this anymore.

```suggestion
    branches:
      - main
```

---

This is removed in aa70fd2662ccae54e66d3fdc14986904b3858a61.

---

This has resurfaced.

---

@MebinAbraham this was needed as the interim PRs against the `feature/datavis` branch would not trigger the CI.

The clean-up is a post-merge task